frontend/src/hooks/@queries/organization.ts

```diff
@@ -16,13 +16,16 @@ type OrganizationResponse = {
   lastAccessed: boolean;
 };
 
-export const useGetOrganizations = () =>
-  useQuery<OrganizationResponse[]>(
+export const useGetOrganizations = () => {
+  const { data: organizations, ...rest } = useQuery<OrganizationResponse[]>(
     [ORGANIZATION_QUERY_KEY.organizations],
     () => getOrganizations(),
     { staleTime: 0, cacheTime: 0 }
   );
 
+  return { organizations: organizations || ([] as OrganizationResponse[]), ...rest };
+};
+
```

Comment:
```suggestion
return {organizations: organizations || [], ...rest};
```

로도 충분하지 않을까요??

아니면 initialValue 적용해줘도 좋을 것 같습니다

Reply:
오우 좋습니다 정상적으로 잘 작동하는군요!

추가적으로 배열의 경우는 undefined 시 이렇게 빈 배열을 내려주도록 하고, 객체의 경우는 그대로 내려주어 사용하는 곳에서 타입 가드를 하는 방식으로 가면 좋을 것 같습니다!
라고 회의 했음

Reply:
좋은 것 같다고 한 것 같은 기억이 나네요 좋습니다.